src/acl.c

```diff
@@ -1981,7 +1981,7 @@ void authCommand(client *c) {
     if (ACLAuthenticateUser(c,username,password) == C_OK) {
         addReply(c,shared.ok);
     } else {
-        addReplyError(c,"-WRONGPASS invalid username-password pair");
+        addReplyError(c,"-WRONGPASS invalid username-password pair, or user has been disabled.");
```

Comment:
"-WRONGPASS invalid username-password pair or user is disabled."

Would be my preference.

Reply:
thanks @madolson , fixed.